.travis.yml

```diff
@@ -221,13 +221,16 @@ matrix:
     os: linux
     env:
     - ARROW_TRAVIS_ORC=1
+    - ARROW_TRAVIS_GANDIVA=1
+    - ARROW_TRAVIS_USE_VENDORED_BOOST=1
     - ARROW_TRAVIS_PARQUET=1
     - BUILD_TORCH_EXAMPLE=no
     - CC="gcc-4.9"
```

Comment:
for some reason, this doesn't work. Can you instead try doing this similar to the C++ gandiva entry ?

- MATRIX_EVAL="CC=gcc-4.9 && CXX=g++-4.9"